<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VehicleDataManagerWindow</class>
 <widget class="QMainWindow" name="VehicleDataManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>679</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>2048</width>
     <height>1024</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QProgressBar" name="transferFileProgressBar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="fileIndexLabel">
        <property name="text">
         <string>( 0 / 0 )</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="fileListWidget"/>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QCheckBox" name="autoRefreshCheckBox">
        <property name="maximumSize">
         <size>
          <width>130</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Auto Refresh</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
